tests/vcs/git/noconflict.py

```diff
@@ -0,0 +1,7 @@
+# A file with merge conflict markers
+print 'Line 1 does something'
+print 'No issues till here'
```

Comment:
Your tests are not passing on the CI. It looks like you have Syntax errors in the test files.
```
==================================== ERRORS ====================================
__________________ ERROR collecting tests/vcs/git/conflict.py __________________
../.pyenv/versions/3.5.1/lib/python3.5/site-packages/py/_path/local.py:662: in pyimport
    __import__(modname)
E     File "/home/ubuntu/coala-bears/tests/vcs/git/conflict.py", line 2
E       print 'Line 1 does something'
E                                   ^
E   SyntaxError: Missing parentheses in call to 'print'
_________________ ERROR collecting tests/vcs/git/noconflict.py _________________
../.pyenv/versions/3.5.1/lib/python3.5/site-packages/py/_path/local.py:662: in pyimport
    __import__(modname)
E     File "/home/ubuntu/coala-bears/tests/vcs/git/noconflict.py", line 2
E       print 'Line 1 does something'
E                                   ^
E   SyntaxError: Missing parentheses in call to 'print'
```